docs/extend/settings.md

```diff
@@ -1,11 +1,11 @@
-#  Settings
+# Settings
 
 At some point while making an extension, you might want to read some of the forum's settings or store certain settings specific to your extension. Thankfully, Flarum makes this very easy.
 
 ## The Settings Repository
 
 Reading or changing settings can be done using an implementation of the `SettingsRepositoryInterface`.
-Because Flarum uses [Laravel's service container](https://laravel.com/docs/container) (or IoC container), you don't need to worry about where to obtain such a repository, or how to instantiate one.
+Because Flarum uses [Laravel's service container](https://laravel.com/docs/container) (or IoC container) for dependency injection, you don't need to worry about where to obtain such a repository, or how to instantiate one.
```

Comment:
```suggestion
Because Flarum uses [Laravel's service container](https://laravel.com/docs/6.x/container) (or IoC container) for dependency injection, you don't need to worry about where to obtain such a repository, or how to instantiate one.
```